modules/alibaba-ai-converting-image-to-qcow2.adoc

```diff
@@ -11,19 +11,30 @@ Convert the generated ISO to `QCOW2` format before importing it into {alibaba}.
 .Prerequisites
 
 * You have created a cluster and downloaded the discovery image in the {ai-full}.
-* You have access to a Red Hat Enterprise Linux or Fedora machine that is outside the cluster, such as your desktop machine.
-* You have ensured that the machine has virtualization flags enabled.
+* You have access to a Linux machine that is outside the cluster, such as your desktop machine. 
 
 .Procedure
 
-. Open the command line interface on your RHEL or Fedora machine.
+. Open the command line interface on the Linux machine.
```

Comment:
Sorry, I missed your question. {op-system-full} gets you Red Hat Enterprise Linux (RHEL) or {op-system-base} gets you RHEL or {op-system-first} gets you Red Hat Enterprise Linux CoreOS (RHCOS)